### PR TITLE
feat(Admin Panel): Possibility to change the theme of the application

### PR DESCRIPTION
**Pull request by MATIFIREofficiel**
Link each component and page with theme.
Admin Panel only for admins
Admin Panel give 9 colors, company name and photo

### CODE REVIEW

File: SurvivorNext/Pages/AdminPanel.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  TextInput,
+  Image,
+  TouchableWithoutFeedback,
+  Button,
+} from 'react-native';
+
+import { useAppContext } from '../AppContext';
+
+export default function AdminPanel() {
+  const buttonColors = ['red', 'orange', 'yellow', 'blue', 'indigo', 'violet', 'purple', 'magenta', 'lime']; // Couleurs des boutons
```
Comment (TurpaultM) [inline SurvivorNext/Pages/AdminPanel.js:18]:
```suggestion
  const buttonColors = ['red', 'orange', 'yellow', 'blue', 'indigo', 'violet', 'purple', 'magenta', 'lime'];
```

Comment (Jibea) [inline SurvivorNext/Pages/AdminPanel.js]:
```suggestion
  const buttonColors = ['red', 'orange', 'yellow', 'blue', 'indigo', 'violet', 'purple', 'magenta', 'lime'];
```

**Review by TurpaultM — APPROVED** [review]

**Review by MathisCaugant — DISMISSED** [review]
Good for me !

**Review by TurpaultM — APPROVED** [review]
great

**Review by Jibea — APPROVED** [review]

**Pull request merged**